src/components/form-error/form-error.js

```diff
@@ -35,8 +35,8 @@ FormError.propTypes = {
 };
 
 FormError.defaultProps = {
-  className: null,
-  message: null,
+  className: undefined,
+  message: undefined,
```

Comment:
is the "undefined" necessary since you already have PropTypes to give you context of what properties are in the class?

Reply:
It's an ESLint rule that enforces to have defaultValues for every non-required prop.

Reply:
I see.  So basically with that Eslint rule, everything IS required?

Reply:
OR does "undefined" breaks "required"

Reply:
It's a couple of ESLINT rules, they counter each other. `isRequired` prop doesn't need a `defaultProp` and every non-`isRequired` prop needs a `defaultProp`.